_includes/v2.2/app/sqlalchemy-large-txns.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy import create_engine, Column, Float, Integer
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+from cockroachdb.sqlalchemy import run_transaction
+from random import random
+
+Base = declarative_base()
```

Comment:
Why are you using the declarative interface here but not in the others? I would expect that if anything you'd be more likely to use the ORM in your main application but the lower-level (and probably more efficient) interfaces in your bulk loader.